src/bci_build/package/python.py

```diff
@@ -97,6 +97,10 @@ def _get_python_kwargs(py3_ver: _PYTHON_VERSIONS, os_version: OsVersion):
         config_sh_script += rf"""; ln -s /usr/bin/python{py3_ver} /usr/local/bin/python3; \
     ln -s /usr/bin/pydoc{py3_ver} /usr/local/bin/pydoc"""
 
+    # broken packaging without a "pip" update-alternatives link
+    if py3_ver in ("3.9",):
+        config_sh_script += rf"""; test -e /usr/bin/pip || ln -s /usr/bin/pip{py3_ver} /usr/local/bin/pip"""
```

Comment:
Do we need the `r` or the multiline string part?
```suggestion
        config_sh_script += f"; test -e /usr/bin/pip || ln -s /usr/bin/pip{py3_ver} /usr/local/bin/pip"
```